### ISSUE TITLE
Bug: Reminder function on weekly check in causing error

### ISSUE THREAD

**Issue by Lostfield** [Friends-of-Monika/mas-selfharm#31]
### What happened?

monika just prompted the first weekly check in, and while she was about to ask me about it, it skipped all dialog and told me there was an error

### Traceback output

```shell
'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/event-handler.rpy", line 3119, in script call
  File "game/submods/Self Harm Awareness Submod/checkup.rpy", line 44, in script
    $ store._msh_reminder.queue_reminder(
  File "game/submods/Self Harm Awareness Submod/checkup.rpy", line 50, in <module>
    grace_period=store._msh_reminder_utils.LATENCY_DAILY
  File "game/submods/Self Harm Awareness Submod/reminder_api.rpy", line 284, in queue_reminder
    __sort_queue()
  File "game/submods/Self Harm Awareness Submod/reminder_api.rpy", line 404, in _m1_reminder_api__sort_queue
    queue.sort(key=lambda reminder: reminder.trigger_at)
TypeError: can't compare datetime.timedelta to datetime.datetime

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/event-handler.rpy", line 3119, in script call
  File "game/submods/Self Harm Awareness Submod/checkup.rpy", line 44, in script
    $ store._msh_reminder.queue_reminder(
  File "C:\Users\ectxB\OneDrive\Desktop\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\ectxB\OneDrive\Desktop\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/submods/Self Harm Awareness Submod/checkup.rpy", line 50, in <module>
    grace_period=store._msh_reminder_utils.LATENCY_DAILY
  File "game/submods/Self Harm Awareness Submod/reminder_api.rpy", line 284, in queue_reminder
    __sort_queue()
  File "game/submods/Self Harm Awareness Submod/reminder_api.rpy", line 404, in _m1_reminder_api__sort_queue
    queue.sort(key=lambda reminder: reminder.trigger_at)
  File "C:\Users\ectxB\OneDrive\Desktop\DDLC-1.1.1-pc\renpy\python.py", line 618, in do_mutation
    return method(self, *args, **kwargs)
TypeError: can't compare datetime.timedelta to datetime.datetime

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.14
```

### Submod version

the anniversary version

### Monika After Story version

0.12.14

### Unstable

No

**Comment by dreamscached** [comment]
Hi, sorry for inconvenience! I've identified an issue and I'll release a fix update for it shortly. For now, you could try pressing 'ignore' button next time this error shows up.

**Comment by Lostfield** [comment]
alright!

**Comment by dreamscached** [comment]
Could you please send your persistent? This would help ensuring the issue is resolved.

**Comment by Lostfield** [comment]
[zip.zip](https://github.com/Friends-of-Monika/mas-selfharm/files/11881340/zip.zip)
i think this is it

**Comment by dreamscached** [comment]
Fixed in 7a1f4e0e013cbb221e7d147fd1d71296aee9eca1, releasing v2.0.2 that fixes this.
